test/e2e/lifecycle/cluster_upgrade.go

```diff
@@ -423,6 +427,35 @@ var _ = SIGDescribe("kube-proxy migration [Feature:KubeProxyDaemonSetMigration]"
 	})
 })
 
+var _ = Describe("[sig-storage] volume mode downgrade [Feature:VolumeModeDowngrade]", func() {
```

Comment:
can we add this to the existing downgrade suite instead of creating a new one?

Reply:
yes, done